lib/src/migrators/module.dart

```diff
@@ -213,29 +224,41 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
     // If a prefix was removed from any members, add an import-only stylesheet
     // that forwards the entrypoint with that prefix.
     if (prefixToRemove != null && renamedMembers.isNotEmpty) {
+      if (importOnlyForAll) {
+        for (var url in migrated.keys.toList()) {
+          if (url == stylesheet.span.sourceUrl) continue;
+          var results = _generateImportOnly(url);
+          if (results != null) migrated[results.item1] = results.item2;
+        }
+      }
       var url = stylesheet.span.sourceUrl;
-      var importOnlyUrl = getImportOnlyUrl(url);
-      var tuple = _absoluteUrlToDependency(url, relativeTo: importOnlyUrl);
-      var results = _generateImportOnly(url, tuple.item1);
-      if (results != null) migrated[importOnlyUrl] = results;
+      var results = _generateImportOnly(url, isEntrypoint: true);
+      if (results != null) migrated[results.item1] = results.item2;
     }
     return migrated;
   }
 
-  /// Generates an import-only stylesheet for [entrypoint] that forwards any
-  /// members that used to have a prefix with that prefix, but forwards other
-  /// members as-is.
+  /// Generates an import-only stylesheet for [url] that forwards any members
+  /// that used to have a prefix with that prefix, but forwards other members
+  /// as-is.
+  ///
+  /// If [url] is an entrypoint, [isEntrypoint] should be true to ensure that
+  /// any members that are forwarded through the entrypoint are also included.
```

Comment:
Why don't we want that forwarding behavior for non-entrypoints? Is this difference tested?

Reply:
`--forward` only forwards members from the entrypoint, not other files, so using the same behavior here for non-entrypoints would cause members from outside the file in question to be considered when determining which members need to be forwarded with a prefix and which don't.

The test I added for this flag fails if you try to use the same behavior for both entrypoints and non-entrypoints b/c this code will think `$unprefixed` is visible from `_prefix` and try to forward based on that.